From 9bbab9bf77365736c8040d70eceb1453bd24b1fc Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Date: Fri, 16 Jan 2026 00:05:37 -0700
Subject: [PATCH 51/76] media: ipu6: isys: Don't set V4L2_FL_USES_V4L2_FH
 manually

The V4L2_FL_USES_V4L2_FH flag is set by v4l2_fh_init(). It is not meant
to be set manually by drivers. Drop it from the ipu6-isys driver.

Signed-off-by: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Reviewed-by: Bingbu Cao <bingbu.cao@intel.com>
Reviewed-by: Sakari Ailus <sakari.ailus@linux.intel.com>
Signed-off-by: Hans Verkuil <hverkuil+cisco@kernel.org>
---
 6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-video.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-video.c b/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-video.c
index 24a2ef9..f3f3bc0 100644
--- a/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-video.c
+++ b/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-video.c
@@ -1306,7 +1306,6 @@ int ipu6_isys_video_init(struct ipu6_isys_video *av)
 	__ipu6_isys_vidioc_try_fmt_meta_cap(av, &format_meta);
 	av->meta_fmt = format_meta.fmt.meta;
 
-	set_bit(V4L2_FL_USES_V4L2_FH, &av->vdev.flags);
 	video_set_drvdata(&av->vdev, av);
 
 	ret = video_register_device(&av->vdev, VFL_TYPE_VIDEO, -1);
-- 
2.43.0

